api/clients/disperser_client.go

```diff
@@ -26,6 +26,14 @@ type Config struct {
 	// TODO: do we want to add config timeouts for those separate requests?
 	Timeout           time.Duration
 	UseSecureGrpcFlag bool
+	// MaxRetrieveBlobSizeBytes is the maximum size of a blob that can be retrieved by using
+	// the RetrieveBlob method. This is used to set the max message size for the grpc client.
+	// DisperserClient uses a single underlying grpc channel shared for all methods,
+	// but all other methods use the default 4MiB max message size, whereas RetrieveBlob
+	// potentially needs a larger size.
+	//
+	// If not set, the default value is 100MiB, to be backward compatible.
```

Comment:
forward compatible

Reply:
It's actually backward compatible. The code before my previous PR (that caused the regression) had a default 100MiB.

Reply:
Ah just saw your previous comment, I see what you mean now! Good point

Reply:
[a07336c](https://github.com/Layr-Labs/eigenda/pull/849/commits/a07336cf2fce85b7d1cf886a3d07f3306ac459b8)